ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.ts

```diff
@@ -51,37 +80,119 @@ import { map, take, filter } from 'rxjs/operators';
                         </section>
                     </ng-container>
                 </ng-container>
-                <section class="section dashboard-section">
+                <section class="section dashboard-section" [class.full-width]="useParticipantDashboard">
                     <div class="content content_medium">
-                        <div class="dashboard-content dashboard-content_table">
-                            <ddp-user-activities [studyGuid]="studyGuid"
-                                                 [displayedColumns]="config.dashboardDisplayedColumns"
-                                                 (open)="navigate($event)">
-                            </ddp-user-activities>
-                        </div>
+                        <mat-accordion *ngIf="useParticipantDashboard; else activitiesTable" hideToggle multi>
+                            <mat-expansion-panel *ngFor="let participant of dashboardParticipants$ | async; first as isFirst"
+                                                 class="dashboard-panel"
+                                                 [expanded]="isFirst"
+                                                 (opened)="openParticipantPanel(participant.userGuid);"
+                                                 (closed)="closeParticipantPanel(participant.userGuid);">
+                                <mat-expansion-panel-header class="dashboard-panel-header">
+                                    <mat-panel-title class="dashboard-panel-title">
+                                        {{participant.label}}
+                                    </mat-panel-title>
+                                    <mat-panel-description fxLayoutAlign="end center" class="dashboard-panel-description">
+                                        <ng-container *ngIf="panelsState.get(participant.userGuid); else show">
+                                            {{'Toolkit.Dashboard.HidePanel' | translate}}
+                                            <mat-icon>expand_less</mat-icon>
+                                        </ng-container>
+                                        <ng-template #show>
+                                            {{'Toolkit.Dashboard.ShowPanel' | translate}}
+                                            <mat-icon>expand_more</mat-icon>
+                                        </ng-template>
+                                    </mat-panel-description>
+                                </mat-expansion-panel-header>
+                                <ng-template matExpansionPanelContent>
```

Comment:
material expancion panel component uses this prefix for identifying lazy loaded content
https://material.angular.io/components/expansion/overview#lazy-rendering
I think it's better using lazy loading here since every activity table component makes api request